app/src/lib/editors/linux.ts

```diff
@@ -11,6 +11,7 @@ export enum ExternalEditor {
   SublimeText = 'Sublime Text',
   Typora = 'Typora',
   SlickEdit = 'SlickEdit',
+  ElementaryCode = 'Code',
```

Comment:
Should this be a more explicit label?

```suggestion
  ElementaryCode = 'Elementary Code',
```

Reply:
I had initially considered that since I think "Code" is more universally used in reference to VS Code, but in the install files, in their the AppCenter, in the menu, and on their github, it's just called  "Code."  Since this is just the user-displayed text in the app, it seemed like it'd be easily recognized to the user that does have it installed by the name "Code," and also seem more of a first-class integration by not inadvertently re-naming their application.

Reply:
👍 these names will also only be visible if the app is installed, so we can assume some familiarity with it if they're on ElementaryOS